app/templates/sectionsummary.html

```diff
@@ -0,0 +1,38 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% extends theme('section_confirmation.html') -%}
+
+{% block page_title -%}{{_("Section-Summary")}} - {{survey_title}}{% endblock -%}
+
+{% block sidebar %}
+  {% if navigation %}
+    {% include theme('partials/navigation.html') %}
+  {% endif %}
+{% endblock %}
+
+{% block form_content %}
+{% endblock %}
+
+{% block main -%}
+
+<div>
+    {% for group in content.summary.groups %}
```

Comment:
This will render multiple titles if there are multiple groups. Presumably we should only ever have one group so the iterator is redundant. And adding multiple titles on top of each other would look odd any way.

Perhaps better to index directly? `{% set group = content.summary.groups | first  %}`

Reply:
Or should we have this title as the section title?

Reply:
Well it is the "Section Summary" so makes sense to use the section title as the title?